components/bl-mapbox/component.json

```diff
@@ -417,6 +434,20 @@
           "label": "Event Data"
         }
       ]
+    },
+    {
+      "id": "setMarkerStatus",
+      "label": "Set Marker Status",
+      "inputs": [
+        {
+          "id": "marker",
+          "label": "Marker"
+        },
+        {
+          "id": "status",
```

Comment:
`Status" true` at screenshots looks not right. Is should be `status: 'active'` or `active: true`